.github/workflows/builds.yml

```diff
@@ -88,6 +88,8 @@ jobs:
       - name: Build (Android)
         if: ${{ contains(matrix.target, 'android') }}
         run: |
+          sudo apt install -y gcc-multilib
+
```

Comment:
```suggestion
```

The FFI builds are already passing, the issues is just the "default" CI builds, feel free to ignore